server/routes/universities.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const router = express.Router();
+
+const University = require('../models/universities');
+require("../models/courses");
+
+
+//GET all universities with populated courses
+router.get("/", async function(req, res, next) {
+    const allUniversities = await University.find()
+    .catch(() => {
+        return res.status(500).send();
+    });
+    res.status(200).send(allUniversities);
+});
+
+//GET university with universityId and populated courses
+router.get('/:universityId/courses', async function(req, res, next) {
+    const universityId = req.params.universityId;
+    const university = await University.findById({ _id: universityId })
+    .populate({path: 'courses', model: 'Course'});
+    res.status(200).send(university);
+});
+
+///////////////////////////////////////////////////////////////////////////
+//// Use these routes to add a university and a course
+///////////////////////////////////////////////////////////////////////////
+
+// router.post('/courses', async function(req, res, next) {
```

Comment:
Is this stuff commented out for any particular reason?  Looks like completed code, so it can be either uncommented or removed if we don't need it.

Reply:
I was using them to make it easier to populate the database with Universities and Courses and thought I'd leave it in if anyone else wanted to add data as well (or until the time we're ready to let users create Universities and Courses).